CustomLogAppender/README.md

```diff
@@ -4,7 +4,7 @@ The appender attaches itself to the desired logs.
 To see the output, you need an error log in the project, which the appender can attach. The error log should be in the process in which you want the appender to append to the log. The log should look like this: ```Ivy.log().error("your log info"); ``` 
 
 ## Integration
-- The custom log appender must contribute the Axon Ivy core bundle `ch.ivyteam.lib.logging`, 
+- The custom log appender has to use the Axon Ivy core bundle `ch.ivyteam.lib.logging`, 
```

Comment:
IMHO; technical detail, but the custom appender actually contributes to the ch.ivyteam.lib.logging bundle, therefore I'd write:
... has to contribute to the AxonIvy core bundle ...

Reply:
has to contribute **to**